public/locales/en/layout/modals/about.json

```diff
@@ -8,6 +8,7 @@
 		"version": "Version",
 		"nodeEnvironment": "Node environment",
 		"i18n": "Loaded I18n translation namespaces",
-		"locales": "Configured I18n locales"
+		"locales": "Configured I18n locales",
+		"experimental_disableEditMode": "*EXPERIMENTAL*: Disable edit mode"
```

Comment:
Why put stars around ? Markdown is not supported aifak. You could just <b/> and use the trans component to achieve bold tho.